### ISSUE TITLE
CARLA Semantic Segmentation

### ISSUE THREAD

**Issue by mhusseinsh** [onlytailei/pytorch-deeplab#2]
Hello @onlytailei 
I downloaded the latest model you committed as you recommended, and downloaded `carla_semantic.pth`, and set it as the pretrained model that the file reads from

I evaluated on 20K CARLA images, and calculated the MSE between the GT and the predicted labels, and it is too much .. it was 73.689, however, this is the saved model that you had for training on CARLA.

Do you have any idea what is the problem ?

I hope to hear from you soon .. thanks
